src/sentry/tasks/data_export.py

```diff
@@ -44,19 +48,28 @@ def assemble_download(data_export_id):
                     file = File.objects.create(
                         name=file_name, type="export.csv", headers={"Content-Type": "text/csv"}
                     )
-                    file.putfile(tf)
+                    file.putfile(tf, logger=logger)
+                    logger.info("dataexport.end", extra={"id": data_export_id})
                     data_export.finalize_upload(file=file)
             except IntegrityError as error:
+                logger.error(
+                    u"dataexport.error: {}".format(error),
```

Comment:
@leeandher If you are going to use string interpolation, please use `six.text_type` so we are Python3 friendly.